Localization/de-DE/home.yaml

```
@@ -41,6 +53,49 @@ Für technische Hilfe: ",
 #URLs - Facebook and Twitter have localised name in certain languages:
 "foldingforum.org",
 "Facebook",
-"Twitter"
+"Twitter",
+
+#LATEST POSTS bold text
+#"LATEST POSTS",
+"NEUESTE ARTIKEL",
+
+# WHY WE NEED YOU bold text
+#"WHY WE NEED YOU",
+"WARUM WIR SIE BRAUCHEN",
+
+# Associated text
+#"Folding@home is a project focused on disease research. The problems we're solving require so many computer calculations and we need your help to find the cures!",
+"Folding@home ist ein Projekt, das sich auf die Erforschung von Krankheiten konzentriert. Die Problemlösungen erfordern eine so große Menge an Berechnungen, dass wir Ihre Hilfe brauchen, um Heilmittel zu finden!",
+
+#FIGHT DESIEASES TOGETHER WITH US bold text
+#"FIGHT DISEASES TOGETHER WITH US.",
+"BEKÄMPFT KRANKHEITEN GEMEINSAM MIT UNS",
+
+#Associated text
+#"The Folding@home software allows you to share your unused computer power with us so that we can research even more potential cures.",
+"Die Folding@home Software erlaubt es Ihnen, uns Ihre ungenutzte Rechenleistung zur Verfügung zu stellen, so dass wir noch mehr mögliche Heilmittel untersuchen können.",
+
+#Associated text in pentagon
+#"1 in a million",
+"1 unter Millionen",
+
+#FOLDING? bold text
+"FALTEN?",
+
+#Associated text
+#"Folding refers to the way human protein folds in the cells that make up your body. We rely on the proteins to keep us healthy and they assemble themselves by folding. But when they misfold, there can be serious consequences to a person's health.",
+"Falten bezieht sich auf die Weise, wie sich menschliche Proteine in den Zellen, aus denen Ihr Körper besteht, verformen. Unsere Gesundheit ist auf Proteine angewiesen, und sie fügen sich durch das Falten zusammen. Aber wenn sie sich falsch falten kann dies ersthafte Folgen für die Gesundheit einer Person haben.",
```

```suggestion
"Falten bezieht sich auf die Weise, wie sich menschliche Proteine in den Zellen, aus denen der menschliche Körper besteht, verformen. Unsere Gesundheit ist auf Proteine angewiesen, und sie fügen sich durch das Falten zusammen. Aber wenn sie sich falsch falten kann dies ersthafte Folgen für die Gesundheit einer Person haben.",
```

in my opinion, it sounds weird with "Ihr" körper in german.

---

an additional comma after "falsch falten"?

---

```suggestion
"Falten bezeichnet den Vorgang, wie sich menschliche Proteine in den Zellen aus denen Ihr Körper besteht, verformen. Unsere Gesundheit ist auf Proteine angewiesen und sie fügen sich durch das Falten zusammen. Wenn sich Proteine jedoch falsch falten, kann dies ersthafte Folgen für die Gesundheit einer Person haben.",
```